nautobot_golden_config/jobs.py

```diff
@@ -97,8 +110,8 @@ def run(self, data, commit):  # pylint: disable=too-many-branches
         """Run config compliance report script."""
         # pylint: disable=unused-argument
 
-        get_refreshed_repos(job_obj=self, repo_type="intended_repository")
-        get_refreshed_repos(job_obj=self, repo_type="backup_repository")
+        get_refreshed_repos(job_obj=self, repo_type="intended_repository", data=data)
```

Comment:
Isn't data = `{}` if there is no inputs selected by the user?  Or is it `None`?

Reply:
Fell victim to forgetting jobs don't auto reload, will verify and swap.

Reply:
The conditional for this never get's hit as the dictionary is populated with keys and values of empty querysets. GOing to investigate a little more. May refactor the query to be more efficient.